source/Playnite/Common/FileSystem.cs

```diff
@@ -348,27 +348,36 @@ public static long GetFileSize(FileInfo fi)
 
         public static long GetDirectorySize(string path)

```

Comment:
This should be imo consolidate with GetDirectorySizeOnDisk and just using an argument, like with GetDirectorySize.